sonar/modules/deposits/cli.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+#
+# Swiss Open Access Repository
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Deposit CLI commands."""
+
+import shutil
+from os import makedirs
+from os.path import exists, join
+
+import click
+from flask import current_app
+from flask.cli import with_appcontext
+from invenio_db import db
+from invenio_files_rest.models import Bucket, FileInstance, Location, \
+    ObjectVersion
+
+
+@click.group()
+def deposits():
+    """Deposits CLI commands."""
+
+
+@deposits.command('create')
+@with_appcontext
+def create():
+    """Create a location and a bucket for uploading files."""
+    click.secho('Creating default location for importing files', bold=True)
+
+    # Directory where files are stored
+    directory = join(current_app.instance_path, 'files')
+
+    if exists(directory):
+        shutil.rmtree(directory)
+
+    makedirs(directory)
+
+    # Remove stored data
+    ObjectVersion.query.delete()
```

Comment:
Is this is not already done by `invenio db destroy create`?

Reply:
Yes, you're right, but the command can be launched alone